x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/CcrLicenseChecker.java

```diff
@@ -134,7 +135,7 @@ public void checkRemoteClusterLicenseAndFetchLeaderIndexMetadataAndHistoryUUIDs(
                     hasPrivilegesToFollowIndices(remoteClient, new String[] {leaderIndex}, e -> {
                         if (e == null) {
                             fetchLeaderHistoryUUIDs(remoteClient, leaderIndexMetaData, onFailure, historyUUIDs ->
-                                    consumer.accept(historyUUIDs, leaderIndexMetaData));
+                                    consumer.accept(historyUUIDs, new Tuple<>(remoteClusterState, leaderIndexMetaData)));
```

Comment:
Don't you need to set `.nodes(true)` on the cluster state request in order to get the minNodeVersion? Why is this not failing any tests?

Reply:
`nodes` defaults to true. But I will set it explicitly.

Reply:
Oh well I see that we set `clear()` which should reset it. I'll investigate why the test did not fail.